client/modules/accounts/components/auth/loginButtons.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component, PropTypes } from "react";
+import {
+  Button
+} from "/imports/plugins/core/ui/client/components";
+
+class LoginButtons extends Component {
+  static propTypes = {
+    capitalizeName: PropTypes.func,
+    currentView: PropTypes.string,
+    loginServices: PropTypes.func,
+    onSeparator: PropTypes.func,
+    onSocialClick: PropTypes.func
+  }
+
+  renderLoginButtons() {
+    const enabledServices = this.props.loginServices().filter((service) =>{
+      return service.enabled;
+    });
+
+    return (
+      <div>
+        {this.props.loginServices &&
+          enabledServices.map((service) => (
+            <Button
+              key={service._id}
+              className={`btn-block provider-${service.name}`}
+              primary={true}
+              bezelStyle="solid"
+              type="button"
+              data-provider={`${service.name}`}
+              onClick={() => this.props.onSocialClick(service.name)}
+            >
+              <i className={`fa fa-${service.name}`} />
+
+              {this.props.currentView === "loginFormSignInView" &&
+                <span data-i18n="accountsUI.signInWith">Sign in with</span>}
+              {this.props.currentView === "loginFormSignUpView" &&
+                <span data-i18n="accountsUI.signUpWith">Sign up with</span>}
+
+              &nbsp;<span data-i18n={`social.${service.name}`}>{this.props.capitalizeName(service.name)}</span>
+          </Button>
+          ))
+        }
+      </div>
+    );
+  }
+
+  renderSeparator() {
+    if (this.props.onSeparator()) {
+      return (
+        <div className="loginForm-seperator">
+          <span />
+          <span className="text" data-i18n="accountsUI.or">or</span>
```

Comment:
ok, we'll correct this.. 

Also @kieckhafer, to speed up review on this... since we're looking at moving fast on it so it can get closed, can run through to you give reviews/comments needed, that way they can be fixed alongside? Also considering the fact that your start of day is our end of day issh